scripts/requirements.txt

```diff
@@ -20,4 +20,6 @@ pymongo>=3.7.2
 pytest>=7.1.2
 scipy>=1.7.3
 statsmodels>=0.13.2
-gnssanalysis>=0.0.28
```

Comment:
Was this intentional? `gnssanalysis` is leveraged in `download_rinex_deps.py`

Reply:
Whoops, not intentional. I had replaced it with my local version of gnssanalysis because I made some changes and they weren't merged in yet. Meant to leave it unchanged in requirements.